README.md

```diff
@@ -50,12 +55,23 @@ In your application, declare the angular-google-analytics module dependency.
 <script src="bower_components/angular-google-analytics/dist/angular-google-analytics.js"></script>
 ```
 
-In your application, declare dependency injection like so.
+#### 2- In your application, declare dependency injection:
 
 ```javascript
-angular.module('myModule', ['angular-google-analytics']);
+var myApp = angular.module('myModule', ['angular-google-analytics']);
 ```
 
+#### 3- Set your Google Analytics account and start tracking:
+
+```JavaScript
+myApp.config(function (AnalyticsProvider) {
+        // Add configuration code as desired
+        AnalyticsProvider.setAccount('UU-XXXXXXX-X');  //UU-XXXXXXX-X should be your tracking code
+    }).run(function(Analytics) {});
```

Comment:
@Toxantron  Sorry, I did not understand what should I do. Could you help me, please?

Do you mean, the bellow code is enough and angular-google-analytics will work?
```
run(['Analytics'], function(Analytics) {
})
```

Where should I add analytics id?

Thanks